doc/lsst.pipe.tasks/index.rst

```diff
@@ -22,6 +22,8 @@ To learn more about the task framework in general, see the :ref:`lsst.pipe.base
 .. .. toctree::
 ..    :maxdepth: 1
 
+   deblending-flags-overview
```

Comment:
From line 15 to line 23 the toctree comes commented out. Please delete the leading `.. ` to remove the comment characters.